helper/testhelpers/testhelpers.go

```diff
@@ -457,3 +457,25 @@ func RaftClusterJoinNodes(t testing.T, cluster *vault.TestCluster) {
 
 	WaitForNCoresUnsealed(t, cluster, 3)
 }
+
+func GenerateDebugLogs(t testing.T, stopCh chan struct{}, client *api.Client) {
+	t.Helper()
+
+	go func() {
+		for {
+			select {
+			case <-stopCh:
```

Comment:
Minor improvement: use a time.Ticker in your select instead of a time.Sleep at the end of your loop.  That way once stopCh is closed we return immediately.

Reply:
Will do

Reply:
You probably have to call `vault.TestWaitActive(t, core.Core)` after the cluster starts so that it finishes setting up before you make any requests.

Reply:
Another alternative to trigger a log line is to mount a secret under a random path. That way you won't need to add that debug log under `handleSysHealthGet`. You can look for `[INFO]  core: successful mount:` so something along those lines.

Reply:
From what I can tell, `testVaultServer` already calls `vault.TestWaitActive`.  Thanks for the suggestion about mounting secrets at a random path.  I'll check that out.

Reply:
Ah, I didn't realize this was referring to `TestMonitorCommand_Run`. For `TestSysMonitorStreamingLogs` you might want to call `TestWaitActive` though (or use `testVaultServer`) since that's using `NewTestCluster`.

Reply:
Looks like `NewTestCluster` already calls this.